### PR TITLE
Update "Club Penguin Rewritten" to 3.0.0

### PR DESCRIPTION
**Pull request by BartBoys3**
- Added separate states for different parts of the login process (Choosing Penguin, Selecting Server, Waiting in Queue)
- Added the ability for party rooms to be identified and reported
image
image
image
image
(The Limo is a room exclusive to the current party, The Penguin Play Awards).

### CODE REVIEW

**Comment by BartBoys3** [comment]
I butchered that commit statement. I changed an else if statement to an if statement

**Comment by doomlerd** [comment]
Please provide screenshots of the changes again.

**Comment by BartBoys3** [comment]
It works the same as last time. Here are new photos regardless. the only changes to the code were removing empty if statements to make DeepScan happy.
image
image
image
image
image
Let me know if you were requesting something else.

File: Club Penguin Rewritten/dist/metadata.json [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
+  "service": "Club Penguin Rewritten",
+  "description": {
+    "en": "A new free virtual world based on Disney's Club Penguin with weeky updates. Waddle around and meet new friends!"
+  },
+  "url": "play.cprewritten.net",
+  "version": "3.0.0",
+  "logo": "https://i.imgur.com/NgaY1xR.jpg",
+  "thumbnail": "https://i.imgur.com/kMkyWlg.png",
+  "color": "#0F4D88",
+  "tags": [
```
Comment (Alanexei) [inline Club Penguin Rewritten/dist/metadata.json:15]: Tags cannot have spaces in them, try replacing them with dashes for example.
https://docs.premid.app/en/dev/presence

**Comment by BartBoys3** [comment]
I didn't intend to dismiss a review. Apparently I did?

**Comment by frozaum** [comment]
nah, it always happens when you do changes, it automatically demiss the last review @BartBoys3

**Pull request merged**